.github/workflows/cd.yml

```diff
@@ -10,4 +10,4 @@ permissions:
 
 jobs:
   release:
-    uses: entur/gha-meta/.github/workflows/release.yml@v1.1.6
+    uses: entur/gha-meta/.github/workflows/release.yml@v1
```

Comment:
Maybe GHA internals should still refer to specific versions?

Reply:
First, it should not happen, if it does, fix the tests :D 
Second a "revert" is quicker, just fix it where the problem is.
Especially now where we have so many changes.. keeping the number of PRs to a minimum is a plus.

Reply:
I like the idea of pinning internal workflows to specific versions, but I agree that it might not be worth the cost of having to maintain a bunch of extra PRs. 

I say we try pinning to major only - if it becomes an issue or limits us somehow, we can always revise and fix it?